### PR TITLE
Pylint Cleanup

### PR DESCRIPTION
**Pull request by bocajnotnef**
Things sanitized:

```
 - oxli/{functions,build_graph}.py
 - scripts/{load-graph,normalize-by-median,abundance-dist}.py
```

things sedated:

```
 - tests/{test_normalize_by_median,test_subset_graph}.py
```

### CODE REVIEW

**Comment by bocajnotnef** [comment]
@mr-c Next victim?

Also, pylint wants to rename `estimate_optimal_with_K_and_M` and the other one--I assume this is because they're terribly long, yes?

File: oxli/functions.py [hunk 1]
```diff
@@ -14,56 +18,57 @@
...
         raise TypeError("K and either M or f must be defined.")
 
 
-def estimate_optimal_with_K_and_M(K, M):
+def estimate_optimal_with_K_and_M(num_kmers, mem_cap):
     """
-    Utility function for estimating optimal counting table args where K is the
-    number of unique kmer and M is the allotted amount of memory
+    Utility function for estimating optimal counting table args where num_kmers
+    is the number of unique kmer and M is the allotted amount of memory
```
Comment (kdmurray91) [inline oxli/functions.py]: @bocajnotnef, ignore my pedantry if it's just that, but `s/M/mem_cap/`?

Comment (bocajnotnef) [inline oxli/functions.py]: Ahh, yes. Good catch, thanks!

File: oxli/functions.py [hunk 2]
```diff
@@ -14,61 +18,63 @@
...
-    if all((K is not None, M is not None, f is None)):
-        return estimate_optimal_with_K_and_M(K, M)
-    elif all((K is not None, M is None, f is not None)):
-        return estimate_optimal_with_K_and_f(K, f)
+    if all((num_kmers is not None, mem_cap is not None, fp_rate is None)):
+        return estimate_optimal_with_K_and_M(num_kmers, mem_cap)
+    elif all((num_kmers is not None, mem_cap is None, fp_rate is not None)):
+        return estimate_optimal_with_K_and_f(num_kmers, fp_rate)
     else:
         raise TypeError("K and either M or f must be defined.")
```
Comment (mr-c) [inline oxli/functions.py]: Shouldn't this message also be updated?

File: oxli/functions.py [hunk 3]
```diff
@@ -92,12 +102,22 @@ def optimal_args_output_gen(unique_kmers, fp_rate):
...
+        to_print.append('{:21e}\t{:19}\t{:17e}\t{:11.3f}'.format(mem_cap,
+                                                                 num_tables,
+                                                                 table_size,
+                                                                 fp_rate))
     return "\n".join(to_print)
 
 
 def do_sanity_checking(args, desired_max_fp):
+    """
+    simple function to check if the restirctions in the args (if there are any)
```
Comment (mr-c) [inline oxli/functions.py]: Did you run codespell?

File: scripts/normalize-by-median.py [hunk 4]
```diff
@@ -243,6 +243,10 @@ def get_parser():
 
 
 def main():  # pylint: disable=too-many-branches,too-many-statements
+    """
```
Comment (mr-c) [inline scripts/normalize-by-median.py]: I'd disable docstrings for main functions of scripts. There is no information here.

File: scripts/normalize-by-median.py [hunk 5]
```diff
@@ -153,6 +150,9 @@ def CatchIOErrors(ifile, out, single_out, force, corrupt_files):
 
 
 def get_parser():
+    """
```
Comment (mr-c) [inline scripts/normalize-by-median.py]: Likewise with get_parser, no value here.

**Comment by bocajnotnef** [comment]
@mr-c Updated!

**Comment by mr-c** [comment]
Looks good. Checklist it and I'll merge

**Comment by bocajnotnef** [comment]
- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Is the Copyright year up to date?

@mr-c Updated!

**Comment by mr-c** [comment]
`make pep257` please

File: khmer/__init__.py [hunk 6]
```diff
@@ -147,7 +147,8 @@ def extract_countinghash_info(filename):
 
 def calc_expected_collisions(hashtable, force=False, max_false_pos=.2):
     """Do a quick & dirty expected collision rate calculation on a hashtable.
-    Check to see that collision rate is within threshold.
+
+    and check to see that collision rate is within threshold.
```
Comment (mr-c) [inline khmer/__init__.py]: "." newline "and ..." ?

**Comment by bocajnotnef** [comment]
@mr-c squashed

**Comment by mr-c** [comment]
Merged; Thanks!

File: tests/test_hashbits.py [hunk 7]
```diff
@@ -1,12 +1,12 @@
 from __future__ import print_function
 from __future__ import absolute_import
 #
-# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
-# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# This file is part of khmer, htabletps://github.com/dib-lab/khmer/, and is
+# Copyrightable (C) Michigan State University, 2009-2015. It is licensed under
```
Comment (ctb) [inline tests/test_hashbits.py]: ahem @mr-c @bocajnotnef.

Comment (bocajnotnef) [inline tests/test_hashbits.py]: Heh.

Sigh.

I'll put the fix in my other pylint PR  #1178

Comment (ctb) [inline tests/test_hashbits.py]: On Wed, Jul 22, 2015 at 08:42:31PM -0700, Jake Fenton wrote:
> > @@ -1,12 +1,12 @@
> >  from __future__ import print_function
> >  from __future__ import absolute_import
> >  #
> > -# This file is part of khmer, https://github.com/dib-lab/khmer/, and is
> > -# Copyright (C) Michigan State University, 2009-2015. It is licensed under
> > +# This file is part of khmer, htabletps://github.com/dib-lab/khmer/, and is
> > +# Copyrightable (C) Michigan State University, 2009-2015. It is licensed under
> 
> Heh.
> 
> Sigh.
> 
> I'll put the fix in my other pylint PR  #1178

No, I've got this fixed in another PR.

Comment (bocajnotnef) [inline tests/test_hashbits.py]: there's also `htableable` stuff down below also.

Comment (ctb) [inline tests/test_hashbits.py]: On Wed, Jul 22, 2015 at 08:52:02PM -0700, Jake Fenton wrote:
> there's also `htableable` stuff down below also.

yes, I think I caught all of it in 3435a5b1d2.

**Pull request merged**